import-dashboards.py

```diff
@@ -231,17 +231,17 @@ def add_panels():
         if os.path.isdir(source_dir):
```

Comment:
While we change this function, a few things:
`for app in ['panels']: ` - this loop looks like rudimental. Can we remove it?


 ```
source_dir = '/usr/share/percona-dashboards/' + app
dest_dir = '/var/lib/grafana/plugins/'
```
^^^ - according to these lines. Can we name these paths as constants (in upper case.)?

Reply:
It has been transformed into a constant.